recipes/parallelproj/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 826a83e51db6459004d17c030c21fb32e7cb5205a45c28c8f50f87f235321196
+
+build:
+  number: {{ build_number }}
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cpu"   # [cuda_compiler_version == "None"]
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cuda"  # [cuda_compiler_version != "None"]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
```

Is the CXX compiler actually used? If not please remove it or add it to the `ignore_run_exports_from` section. So that your library doesn't pick up the CXX standard library as a dependency.

EDIT: Looks like it is not used for non-cuda, so do this conditionally.

```suggestion
    - {{ compiler('cxx') }}  # [cuda_compiler_version != "None"]
```